cibuildwheel/macos.py

```diff
@@ -14,11 +14,11 @@
 def get_python_configurations(build_selector):
     PythonConfiguration = namedtuple('PythonConfiguration', ['version', 'identifier', 'url'])
     python_configurations = [
-        PythonConfiguration(version='2.7', identifier='cp27-macosx_10_6_intel', url='https://www.python.org/ftp/python/2.7.17/python-2.7.17-macosx10.6.pkg'),
-        PythonConfiguration(version='3.5', identifier='cp35-macosx_10_6_intel', url='https://www.python.org/ftp/python/3.5.4/python-3.5.4-macosx10.6.pkg'),
-        PythonConfiguration(version='3.6', identifier='cp36-macosx_10_6_intel', url='https://www.python.org/ftp/python/3.6.8/python-3.6.8-macosx10.6.pkg'),
-        PythonConfiguration(version='3.7', identifier='cp37-macosx_10_6_intel', url='https://www.python.org/ftp/python/3.7.5/python-3.7.5-macosx10.6.pkg'),
-        PythonConfiguration(version='3.8', identifier='cp38-macosx_10_9_x86_64', url='https://www.python.org/ftp/python/3.8.0/python-3.8.0-macosx10.9.pkg'),
+        PythonConfiguration(version='2.7', identifier='cp27-macosx_intel', url='https://www.python.org/ftp/python/2.7.17/python-2.7.17-macosx10.6.pkg'),
+        PythonConfiguration(version='3.5', identifier='cp35-macosx_intel', url='https://www.python.org/ftp/python/3.5.4/python-3.5.4-macosx10.6.pkg'),
+        PythonConfiguration(version='3.6', identifier='cp36-macosx_intel', url='https://www.python.org/ftp/python/3.6.8/python-3.6.8-macosx10.6.pkg'),
+        PythonConfiguration(version='3.7', identifier='cp37-macosx_intel', url='https://www.python.org/ftp/python/3.7.5/python-3.7.5-macosx10.6.pkg'),
+        PythonConfiguration(version='3.8', identifier='cp38-macosx_x86_64', url='https://www.python.org/ftp/python/3.8.0/python-3.8.0-macosx10.9.pkg')
```

Comment:
Ha, funny. So Python has not adopted the new "macOS" name yet, but is still using `macosx`? Are we sure this is not going to change too soon?

Reply:
You see that this platform tag is used in installer name. And it is default name given by setuptools.

Reply:
I know. But there is no plan to change this soon, is there?

Reply:
I know. But there is no plan to change this soon, is there?

Reply:
I did not come across anything suggesting this. IMHO, I think it would be a nightmare to change this and Python/CPython/PyPA will probably stick with macosx (at least as long as possible). The amount of libraries failing with such a change would be huge and carefully communicated during BETA/RC phases.

> PS: Will PyPI allow you to upload wheels with newer macOS versions, like '{package_name}-{package_version}-cp37-cp37m-macosx_10_14_intel.whl' for example?

Short answer , yes. Long answer:
https://github.com/pypa/warehouse/blob/a37ea8d9e06361688a498fbcb83015c58fd9150d/warehouse/forklift/legacy.py#L116-L138

Reply:
OK, great! Good to know, thanks :-)